gitautosync/__init__.py

```diff
@@ -131,7 +139,14 @@ def _pull_and_resolve_conflicts(self):
 
         yield from execute_cmd(['git', 'checkout', self.branch_name], cwd=self.repo_dir)
         yield from execute_cmd(['git', 'fetch'], cwd=self.repo_dir)
-        yield from execute_cmd(['git', 'merge', '-Xours', 'origin/{}'.format(self.branch_name)], cwd=self.repo_dir)
+
+        try:
+            yield from execute_cmd(['git', 'merge', '-Xours', 'origin/{}'.format(self.branch_name)], cwd=self.repo_dir)
+        except subprocess.CalledProcessError:
+            # Keep the modified file if a change to it is made
```

Comment:
This should explicitly figure out *why* this failed, and if this is the right thing to do. Shouldn't do this blanket on CalledProcessError.

Reply:
Also can you write a longer comment explaining how this fixes the issue? I don't quite get it yet.

Reply:
I put a more detailed explanation of how this fixes it in the new commit. 

The try catch is to see if there will be a merge error b/c of differences in statuses of the files in the origin vs locally. I thought doing a try catch was was the cleanest way to do it since to figure out if there was a merge error another way would involve using additional git commands and having to parse those to see if there is an error, which i thought would add more complexity. However, I tried narrowing down the catch so that it only catches the errors with an exist status of 1. Lmk if you think this is sufficient. If not, I'll try looking into an alternate solution.